.github/workflows/conformance-eks-v1.11.yaml

```
@@ -104,6 +105,11 @@ jobs:
           filters: |
             src:
               - '!(test|Documentation)/**'
+            eni:
+              - 'pkg/aws/**'
+              - 'pkg/ipam/**'
+              - 'pkg/policy/groups/aws/**'
+              - 'vendor/github.com/aws/aws-sdk-go-v2/**'
```

Nit: There's also `github.com/aws/smithy-go` which seems to be used in `pkg/aws/eni`, so maybe we should match this more generically:

```suggestion
              - 'vendor/github.com/aws/**'
```